content/redirects.txt

```diff
@@ -136,8 +136,9 @@
 
 
 # Extending Terraform section
-/guides/writing-custom-terraform-providers.html          /docs/extend/writing-custom-providers.html
+/guides/writing-custom-terraform-providers.html          https://learn.hashicorp.com/terraform/providers/setup-implement-read
```

Comment:
These need to be commented out! They're just there as a record of intent, but we need to make sure the thingy doesn't actually try to parse them. 

```suggestion
# /guides/writing-custom-terraform-providers.html          https://learn.hashicorp.com/terraform/providers/setup-implement-read
```